workflow for developing zkSNARK dApp by zokrates on bitcoin/README.md

```diff
@@ -0,0 +1,71 @@
+# 比特币上基于 zokrates 的 zkSNARK 应用开发流程简介
+
+
+我们展示了[如何使用 zk-SNARKs 来开发比特币的信息不完整的游戏](https://xiaohuiliu.medium.com/incomplete-information-games-on-bitcoin-d79408050882)。
+
+
+众所周知，[ZoKrates](https://github.com/Zokrates/ZoKrates) 是 zkSNARKs 的工具箱。它可以帮助您在 DApp 中使用可验证的计算，从规范的基于高级语言的程序电路到生成计算证明，再到验证这些证明。
+
+
+在 [ZoKrates Fork 库](https://github.com/sCrypt-Inc/zokrates) 中，我们扩展了 ZoKrates 的功能。使其支持自动生成验证 zk-SNARKs 证明的sCrypt智能合约。
+
+
+## 安装
+
+下载并使用我们发布的二进制包:
+
+https://github.com/sCrypt-Inc/zokrates/releases/tag/v0.8.1
+
+## 开发流程
+
+开发流程基本与原生的 ZoKrates 相同。在验证证明阶段会有不同。
+
+1. 编写电路程序，创建文本文件 `root.zok` 并实现一下的程序。在这个例子中，我们将证明知道数字 `b` 的平方根为 `a` ：
+
+```
+def main(private field a, field b) {
+    assert(a * a == b);
+    return;
+}
+```
+
+2. 编译电路
+
+```
+zokrates compile -i root.zok
+```
+
+3. 执行设置
+
+```
+zokrates setup
+```
+
+4. 计算见证人
+
+```
+zokrates compute-witness -a 337 113569
+```
+
+5. 生成证明
+
+```
+zokrates generate-proof
+```
+
+6. 导出验证者智能合约 `verifier.scrypt`, 同时会提供一个 `verifier.js` 文件
+
+```
+zokrates export-verifier-scrypt
+```
+
+7. 使用第六步导出的 `verifier.js` 在本地执行对证明的验证。如果本地验证成功，则会将 `verifier.scrypt` 验证者合约部署到测试网(需要填写测试网私钥)，并调用部署的验证者合约
+
+```
+node --max-old-space-size=8192 verifier.js
```

Comment:
Just `node verifier.js`.

Put this in a footnote, only needed if node memory runs out.
--max-old-space-size=8192

Reply:
it   must out of memory. So put it in the most visible position to prevent him from making mistakes the first time